Specs/Scene/PolygonClippingAccelerationGridSpec.js

```diff
@@ -3,6 +3,7 @@ import PolygonClippingAccelerationGrid from "../../Source/Scene/PolygonClippingA
 describe("Scene/PolygonClippingAcceleration", function () {
   it("developer error if invalid number of positions or indicies provided", function () {
     expect(function () {
+      // eslint-disable-next-line no-unused-vars
       var empty_arrays = new PolygonClippingAccelerationGrid({
```

Comment:
This is fine, but as a tip, you can also just do `return` instead of `var empty_arrays =` and the test will do what you want without worrying about eslint or local vars.

Reply:
Also, `empty_arrays` should probably be `emptyArrays`.  We use camel case for variable names.